operator/pkg/apis/cassandra/v1beta1/cassandradatacenter_types.go

```diff
@@ -219,6 +219,9 @@ type CassandraDatacenterSpec struct {
 	// Cassandra users to bootstrap
 	Users []CassandraUser `json:"users,omitempty"`
 
+	// Preserve client source-ips
+	PreserveClientSourceIps bool `json:"preserveClientSourceIps,omitempty"`
```

Comment:
maybe something like

```suggestion
	NodePort *int `json:"nodePort,omitempty"`
```

where they can set it to 0 to get an auto-assigned port, or any number to request a specific port?

Reply:
oh maybe we need two ports on the node port service? 🤦 one to replace 9042 and another to replace 7000 (or 7001)